04-onion-routing.md

```diff
@@ -541,6 +541,14 @@ setting for the outgoing channel:
 1. type: UPDATE|14 (`expiry_too_soon`)
 2. data:
    * [`92`:`channel_update`]
+   
+If the channel is disabled, we tell them the the current channel
+setting for the outgoing channel:
+   
+1. type: UPDATE|20 (`channel_disabled`)
+2. data:
+   * [`2`: `flags`]
+   * [`92`:`channel_update`]
```

Comment:
I don't see it as a temporary failure: disabling a channel (e.g. when the peer is disconnected) is similar to updating the fee: we publish a `channel_update`, and from now on we will send `channel_disabled` in response to htlcs attempting to use the channel.